diff --git a/node_modules/pear-runtime-updater/index.js b/node_modules/pear-runtime-updater/index.js
index adb3fd3..c081633 100644
--- a/node_modules/pear-runtime-updater/index.js
+++ b/node_modules/pear-runtime-updater/index.js
@@ -18,7 +18,7 @@ module.exports = class PearRuntimeUpdater extends ReadyResource {
     if (!opts.dir) throw new Error('dir required')
     if (!opts.upgrade) throw new Error('upgrade link required')
     if (!opts.name) throw new Error('name required')
-
+      console.log(opts, 'constructor opts')
     this.dir = opts.dir
     this.version = opts.version || 0
     this.app = opts.app
@@ -98,6 +98,7 @@ module.exports = class PearRuntimeUpdater extends ReadyResource {
       const manager = new MSIXManager()
       await manager.addPackage(nextApp, { restartOnUpdate: this.win32RestartAfterUpdate })
     } else {
+      console.log(nextApp, this.app, 'swap 1)nextApp 2)thisAPp')
       await fsx.swap(nextApp, this.app)
     }
     await fs.promises.rm(this.next, { recursive: true, force: true })
@@ -114,11 +115,14 @@ module.exports = class PearRuntimeUpdater extends ReadyResource {
     const length = this.drive.core.length
     const id = length + '.' + this.drive.core.fork
     const next = path.join(this.dir, 'pear-runtime/next', id)
+    console.log( '_update next', next)
+    console.log('dir', this.dir)
     const co = this.drive.checkout(length)
 
     this.checkout = co
 
     const manifest = await co.get('/package.json')
+    console.log(JSON.parse(manifest).version, this.version, 'manifest version, this.version')
     if (!manifest || JSON.parse(manifest).version === this.version) {
       this.updating = false
       this.checkout = null
@@ -129,8 +133,11 @@ module.exports = class PearRuntimeUpdater extends ReadyResource {
     const local = new Localdrive(next)
 
     this.emit('updating')
+    console.log("updating")
     const prefix = `/by-arch/${host}/app/${this.name}`
+    console.log('prefix', prefix)
     for await (const data of co.mirror(local, { prefix })) {
+      console.log('data', data)
       this.emit('updating-delta', data)
     }
 
@@ -143,6 +150,7 @@ module.exports = class PearRuntimeUpdater extends ReadyResource {
 
     this.updating = false
     this.updated = true
+    console.log('updated')
     this.emit('updated')
 
     if (this.drive.core.length > length) this._updateBackground()
